src/lib/y2storage/proposal/autoinst_space_maker.rb

```diff
@@ -72,42 +73,55 @@ def cleaned_devicegraph(original_devicegraph, drives_map, planned_devices)
 
       # Deletes unwanted partitions for the given disk
       #
-      # @param devicegraph  [Devicegraph]
-      # @param disk         [Disk]
-      # @param drive_spec   [AutoinstProfile::DriveSection]
-      # @param reused_parts [Array<String>] Reused partitions names
-      def delete_stuff(devicegraph, disk, drive_spec, reused_parts)
-        reused_parts ||= []
-        if drive_spec.initialize_attr && reused_parts.empty?
+      # @param devicegraph    [Devicegraph]
+      # @param disk           [Disk]
+      # @param drive_spec     [AutoinstProfile::DriveSection]
+      # @param reused_devices [Array<String>] Reused disks and partitions names
```

Comment:
I would document that this method allows nil ( as later ones that use identical parameter reused_devices crash if nil is passed ).